atom/common/api/event_emitter_caller.cc

```diff
@@ -20,8 +20,13 @@ v8::Local<v8::Value> CallMethodWithArgs(v8::Isolate* isolate,
                                    v8::MicrotasksScope::kRunMicrotasks);
   // Use node::MakeCallback to call the callback, and it will also run pending
   // tasks in Node.js.
-  return node::MakeCallback(isolate, obj, method, args->size(), &args->front(),
-                            {0, 0}).ToLocalChecked();
+  v8::MaybeLocal<v8::Value> ret = node::MakeCallback(isolate, obj, method,
+                                                     args->size(),
+                                                     &args->front(), {0, 0});
```

Comment:
Question -- not part of the review, as this isn't new to your patch anyway -- why do we use `{0, 0}` instead of `{}`? Do we support a compiler that doesn't support uniform initialization?

Reply:
`{0, 0}` is used in Node's source code, we are following their style.

Reply:
Makes sense. Thanks for the answer :)